.github/workflows/publish.yml

```diff
@@ -0,0 +1,85 @@
+name: publish
```

Comment:
It's more like `cd: add publish workflow `

Reply:
Fixed